mentat/app.py

```diff
@@ -205,8 +205,16 @@ def user_filter_changes(
     indices = []
     for index, change in enumerate(code_changes, start=1):
         print_change(change)
+        # Allowing the user to remove rename file changes introduces a lot of edge cases
```

Comment:
What edge cases have you discovered/thought about so far?

One that comes to mind is a file being renamed and imports not being updated accordingly.

Reply:
I was thinking more edge cases in how we handle it; the big one being that if we don't rename the file, then any changes the model suggests that try to change the renamed file will be trying to change a non-existent file. It gets worse if the model is renaming multiple files to similar names, or one file twice, and so on.